Combinatorial_map/include/CGAL/Combinatorial_map_storages.h

```diff
@@ -405,5 +411,9 @@ namespace CGAL {
 
 } // namespace CGAL
 
+
+#if  (BOOST_GCC >= 40800)
+ _Pragma("GCC diagnostic pop")
+#endif
 #endif // CGAL_COMBINATORIAL_MAP_H //
```

Comment:
The diagnostic options are pushed for gcc>=5.0 but popped for gcc>=4.8! That will probably do something strange for 4.8<=gcc<5.